### PR TITLE
readme: add functional test dependencies

### PR DESCRIPTION
**Pull request by devimc**
add functional test dependencies and installation instructions

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
lgtm, thanks @devimc for adding the missing dependencies

File: README.rst [hunk 1]
```diff
@@ -3,10 +3,22 @@ Tests
...
+
+- github.com/onsi/ginkgo/ginkgo
+- github.com/onsi/gomega
+
+To install above packages, just run::
+
+  $ go get github.com/onsi/ginkgo/ginkgo
+  $ go get github.com/onsi/gomega
+
+and add ``$GOPATH/bin`` to the ``$PATH``
```
Comment (chavafg) [inline README.rst]: What about adding the instruction: 
```
export GOBIN=$GOPATH/bin
export PATH=$PATH:$GOBIN
```

Comment (sboeuf) [inline README.rst]: Agreed, this would be better. It's enough generic to be written down into the readme.

File: README.rst [hunk 2]
```diff
@@ -3,10 +3,22 @@ Tests
 
 Clear Containers Tests Repository
 
-Running functional tests
+Functional tests
 ------------------------
 
-To run functional tests::
+In order to run all functional test you have to install next packages
```
Comment (dlespiau) [inline README.rst]: you shouldn't need to do that, that's the purpose of vendoring. The git clone needs to be inside $GOROOT though.

File: README.rst [hunk 3]
```diff
@@ -3,10 +3,22 @@ Tests
 
 Clear Containers Tests Repository
 
-Running functional tests
+Functional tests
```
Comment (jodh-intel) [inline README.rst]: For rst, the underline for a heading must be the same length as the heading itself, so the line below needs to be of length `strlen("Functional tests")` :smile:

Comment (devimc) [inline README.rst]: oops fixing, thanks

File: README.rst (hunk 2 above)
Comment (devimc) [inline README.rst]: yes, but you need the binary ```ginkgo``` in the PATH

**Comment by chavafg** [comment]
lgtm

**Pull request merged**